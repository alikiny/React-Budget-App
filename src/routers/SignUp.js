import React from 'react';
import TextField from '@material-ui/core/TextField';



const SignUp = ({signUpAction}) => (
    <form className="form">
        <TextField
            label="Email"
            id='email'
            value=''
            onChange={() => console.log()}
            variant="outlined"
        >
        </TextField>


        <TextField
            label="User name"
            id='user-name'
            value=''
            onChange={() => console.log()}
            variant="outlined"
        >
        </TextField>


        <TextField
            label="Password"
            id='password'
            value=''
            onChange={() => console.log()}
            variant="outlined"
        >
        </TextField>


        <TextField
            label="Retype-password"
            id='password'
            value=''
            onChange={() => console.log()}
            variant="outlined"
        >
        </TextField>

        <button type="submit" onClick={signUpAction}> Register </button>

        <p>Already have an account? 
        <a >Log in</a></p>

    </form>

)


export default SignUp