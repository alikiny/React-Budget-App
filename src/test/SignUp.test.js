import React from 'react'
import SignUp from '../routers/SignUp'
import {shallow} from 'enzyme'

test('LogIn page render corrently',()=>{
    const wrapper=shallow(<SignUp signUpAction={()=>{}}/>)
    expect(wrapper).toMatchSnapshot()
})