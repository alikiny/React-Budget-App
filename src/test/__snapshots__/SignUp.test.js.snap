// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogIn page render corrently 1`] = `
<form
  className="form"
>
  <WithStyles(ForwardRef(TextField))
    id="email"
    label="Email"
    onChange={[Function]}
    value=""
    variant="outlined"
  />
  <WithStyles(ForwardRef(TextField))
    id="user-name"
    label="User name"
    onChange={[Function]}
    value=""
    variant="outlined"
  />
  <WithStyles(ForwardRef(TextField))
    id="password"
    label="Password"
    onChange={[Function]}
    value=""
    variant="outlined"
  />
  <WithStyles(ForwardRef(TextField))
    id="password"
    label="Retype-password"
    onChange={[Function]}
    value=""
    variant="outlined"
  />
  <button
    onClick={[Function]}
    type="submit"
  >
     Register 
  </button>
  <p>
    Already have an account?
    <a>
      Log in
    </a>
  </p>
</form>
`;
